docs/metrics/lead-cycle-time.md

```diff
@@ -0,0 +1,33 @@
+# Lead and cycle time metrics
+
+Monitoring you team's lead time and cycle time allows you to understand if you're improving the ability to deliver value to customers.
+
+![Lead time versus cycle time](images/lead-cycle-time.png)
+
+-   [Lead time](#lead-time)
+-   [Cycle time](#cycle-time)
+
+!!! note
+    Pulse calculates lead time and cycle time based on the state changes of issues **that are already closed** in Jira. This means that:
+
+    -   **If you reopen** a completed issue, Pulse stops considering the issue while calculating the metrics. If the issue is completed again, Pulse takes into account the last completed timestamp.
+
+    -   **If you change the type or the project** of a completed issue, Pulse reflects the change on the dashboards, transitioning the issue history to the new project if relevant
+    
+        Note that this change doesn't affect the overall calculation of the lead time or cycle time metrics.
+
+## Lead time
+
+Time between creating an issue in the backlog and completing the issue.
+
+```text
+issue completed timestamp - issue created timestamp
+```
+
+## Cycle time
+
+Time between starting to work on an issue and completing the issue. Cycle time is a subpart of lead time.
+
+```text
+issue completed timestamp - issue in progress timestamp
```

Comment:
I don't think it's as simple as this, since an issue can be moved between 'In Progress' and and 'To Do' any number of times.

However, it's not easy to have a simple formula to compute cycle time as I define it.

Reply:
How are we calculating the cycle time? I can think of three options:

* last issue completed timestamp - **first** issue in progress timestamp
* last issue completed timestamp - **last** issue in progress timestamp
* sum of all pairs (issue completed timestamp - issue in progress timestamp)

Reply:
> sum of all (issue completed timestamp - issue in progress timestamp) pairs

It's this one.

Reply:
Thanks @afsmeira, it makes sense!

I added some more detail to this page, making it more explicit that Pulse takes into account issues that transition to a certain status such as **Completed** and **In progress** more than once.